src/Table.luna

```diff
@@ -152,21 +380,100 @@ class Table:
+    #
+    # `return`: `Table` value without rows containing NA records.
+
     def dropNa:
         self.fromWrapper $ self.ptr.dropNa
+
+    # Remove rows where missing values occurs for given column.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,,5]
+    # > l2 = [11,,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > newTable = table.dropNaAt "col2"
+    #
+    # `name`: Column name for which we check NA values.
+    # `return`: `Table` value without rows containing NA records in given column.
+
     def dropNaAt columnName:
         self.fromWrapper $ self.ptr.dropNaByName columnName
+
+    # Fill NA/NaN values with `valueToFillWith`.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,,5]
+    # > l2 = [11,,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > newTable = table.fillNa 53
+    #
+    # `valueToFillWith`: Value to use to fill holes (e.g. 0).
+    # `return`: `Table` with holes filled by `valueToFillWith`.
+
     def fillNa valueToFillWith:
         self.fromWrapper $ self.ptr.fillNa valueToFillWith
+
+    # Fill NA/NaN values with `valueToFillWith` in column `columnName`
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,,5]
+    # > l2 = [11,,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > newTable = table.fillNaAt "col1" 53
+    #
+    # `columnName`: Name of the column to fill holes in.
+    # `valueToFillWith`: Value to use to fill holes (e.g. 0).
+    # `return`: `Table` with holes in column `columnName` filled with `valueToFillWith `.
+
     def fillNaAt columnName valueToFillWith:
         self.fromWrapper $ self.ptr.fillNaAt columnName valueToFillWith
+
     def interpolate:
         self.fromWrapper $ self.ptr.interpolate
     def toColumn:
         case self.toList of
             [c]: c
             t: throw $ "Table.toColumn: table must have exactly 1 column, but has " + t.length.toText
 
+    # Generates `Table` with number and ratio of NA/NaN values for each column.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,,5]
+    # > l2 = [11,,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > newTable = table.describeNa
+    #
+    # `return`: `Table` value with statistics of NA/NaN values for each column.
```

Comment:
What are the names of columns in `return`?

Reply:
"ratio" -> "percentage" :)